chassis/invokers/src/main/java/com/griddynamics/jagger/invoker/http/v2/JHttpQuery.java

```diff
@@ -0,0 +1,372 @@
+package com.griddynamics.jagger.invoker.http.v2;
+
+import org.springframework.http.HttpHeaders;
```

Comment:
It's just a value object with getters and setters, nothing more. We can create a copy of this class  and put it to our code base, but it has no sense, I think.